src/sentry/integrations/vsts/client.py

```diff
@@ -229,3 +232,35 @@ def get_users(self, account_name):
             ),
             api_preview=True,
         )
+
+    def create_subscription(self, instance, external_id):
+        return self.post(
+            VstsApiPath.subscriptions.format(
+                account_name=instance
+            ),
+            data={
+                'publisherId': 'tfs',
+                'eventType': 'workitem.updated',
+                'resourceVersion': '1.0',
+                'consumerId': 'webHooks',
+                'consumerActionId': 'httpRequest',
+                'consumerInputs': {
+                    'url': absolute_uri('/extensions/vsts/webhook/'),
+                    'resourceDetailsToSend': 'all',
+                }
+            },
+        )
+
+    def delete_all_subscriptions(self, instance):
+        subscriptions = self.get(VstsApiPath.subscriptions.format(
+            account_name=instance
+        ))
+        delete_url = 'https://{account_name}/_apis/hooks/subscriptions/{subscription_id}?api-version=4.1'
```

Comment:
Should this go up with the other URLs?

Reply:
I wasn't sure whether to keep that code or remove it. I mostly used it for testing.